### PR TITLE
update task lock docs

### PR DESCRIPTION
**Pull request by mski-iksm**
I've updated the docs of redis task lock.

Please review.

### CODE REVIEW

**Review by hirosassa — APPROVED** [review]
Looks great improvement! Thanks!

**Comment by hirosassa** [comment]
@mski-iksm I merged https://github.com/m3dev/gokart/pull/264, so please rebase it.

**Comment by mski-iksm** [comment]
> @mski-iksm I merged #264, so please rebase it.

@hirosassa 
I've rebased the newest master. Please check!

File: docs/using_task_cache_collision_lock.rst [hunk 1]
```diff
@@ -75,11 +101,12 @@ Settings in this section will prevent running ``run()`` at the same time for eff
         redis_port=6379
         redis_fail_on_collision=true
 
-4. Set retry parameters. Set following parameters to retry when task
-   failed. Values of ``retry_count`` and ``retry_delay``\ can be set to
-   any value depends on your situation.
+4. Set retry parameters
+
+    Set following parameters to retry when task failed.
+    Values of ``retry_count`` and ``retry_delay`` are set in seconds.
```
Comment (kitagry) [inline docs/using_task_cache_collision_lock.rst:107]: I think `retry_count` is not set in seconds.

I like the listing, but feel free.

```suggestion
    * ``retry_count``: the max number of retries
    * ``retry_delay``: this value is set in seconds
```

**Review by Hi-king — APPROVED** [review]

**Comment by Hi-king** [comment]
@mski-iksm thx for maintaining good docs :)

**Pull request merged**